options/models.py

```diff
@@ -24,6 +24,9 @@ class Option(models.Model):
     value = models.TextField()
     value_type = models.CharField(max_length=8, default=TYPE_TXT, choices=TYPE_CHOICES)
 
+    # whether this option is public(true) or private(false)
```

Comment:
I think this field is pretty self-descriptive, so you can probably just delete lines 26 and 27